t/001-tls.t

```diff
@@ -457,3 +457,41 @@ nil, connection is not TLS or TLS support for Nginx not enabled
 [alert]
 [warn]
 [crit]
+
+
+
+=== TEST 7: ssl.get_ssl_pointer
+--- http_config
+    lua_package_path "../lua-resty-core/lib/?.lua;lualib/?.lua;;";
+    lua_ssl_protocols SSLV3 TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Should we use sslv3 or tlsv1?

Reply:
It is just to keep consistent with the previous test configuration.